recipes/nltk-data/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{%set name = "nltk" %}
+{%set version = "2017.10.22" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "14b55df9314ece2e7e53258c41b2f29a1fe0d7517ded586fa9ed137edf9a39ec" %}
+
+package:
+  name: {{ name }}-data
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://github.com/nltk/nltk_data/archive/gh-pages.zip
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  build:
+    - unzip
+  run:
+    - python
+    - nltk >=3.2.5
+    - six
+
+test:
+  imports:
+    - nltk
+  files:
+    - test_data.py
+  commands:
+    - python test_data.py
+
+about:
+  home: http://nltk.org/
+  license: Apache 2.0
```

Comment:
Add `license_family: Apache`.

There should also be a `license_file`, but it's actually not clear to me what license this data package is under. The included corpora are presumably under a variety of licenses, and there's no explicit license file in this directory or – as far as I can tell – any reason to think that the NLTK license actually applies to this separately-downloaded stuff....

Reply:
https://github.com/nltk/nltk_data/issues/102